extensions/pkg/terraformer/config.go

```diff
@@ -42,8 +42,25 @@ const (
 )
 
 // SetVariablesEnvironment sets the provided <tfvarsEnvironment> on the Terraformer object.
+// Deprecated: use SetEnvVars instead
 func (t *terraformer) SetVariablesEnvironment(tfvarsEnvironment map[string]string) Terraformer {
-	t.variablesEnvironment = tfvarsEnvironment
+	var envVars []corev1.EnvVar
+	for k, v := range tfvarsEnvironment {
+		envVars = append(envVars, corev1.EnvVar{Name: k, Value: v})
+	}
+
+	// maps are unsorted, sort resulting slice after looping over map to avoid flickering results
+	sort.SliceStable(envVars, func(i, j int) bool {
```

Comment:
Nitpick: I would think that you don't need stable sorting since you are sorting on map keys and those should be unique

Reply:
Well, that's a nice catch 😄
You're right, so I changed it to `sort.Slice`.